.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 
 node_js:
-  - '0.10'
```

Comment:
That's why I tend to do stuff like https://github.com/pascalduez/postcss-apply/blob/master/test/control.test.js#L7
I know, a bit sad. But should be removed soonish.